src/index.ts

```diff
@@ -35,76 +35,150 @@ export interface BackgroundProcessorOptions extends ProcessorWrapperOptions {
   onFrameProcessed?: (stats: FrameProcessingStats) => void;
 }
 
+/**
+ * Instantiates a background processor that is configured in blur mode.
+ * @deprecated Use `BackgroundProcessorManager.withBackgroundBlur` instead.
+ */
 export const BackgroundBlur = (
   blurRadius: number = 10,
   segmenterOptions?: SegmenterOptions,
   onFrameProcessed?: (stats: FrameProcessingStats) => void,
   processorOptions?: ProcessorWrapperOptions,
 ) => {
-  return BackgroundProcessor(
-    {
-      blurRadius,
-      segmenterOptions,
-      onFrameProcessed,
-      ...processorOptions,
-    },
-    'background-blur',
+  const processor = BackgroundProcessorManager.withBackgroundBlur(
+    blurRadius,
+    segmenterOptions,
+    onFrameProcessed,
+    processorOptions,
   );
+  processor.name = 'background-blur';
+  return processor;
 };
 
+/**
+ * Instantiates a background processor that is configured in virtual background mode.
+ * @deprecated Use `BackgroundProcessorManager.withVirtualBackground` instead.
+ */
 export const VirtualBackground = (
   imagePath: string,
   segmenterOptions?: SegmenterOptions,
   onFrameProcessed?: (stats: FrameProcessingStats) => void,
```

Comment:
The old `VirtualBackground` / `BackgroundBlur` functions still exist but are marked as `@deprecated` now. I'm thinking they could be removed in the next major version.

Reply:
Eventhough it's technically not necessary for pre 1.0 releases, I'd still argue we should make this PR a minor version update